dspace-server-webapp/src/test/java/org/dspace/app/rest/ResourcePolicyRestRepositoryIT.java

```diff
@@ -0,0 +1,578 @@
+        Community community2 = CommunityBuilder.createCommunity(context).withName("My community_2").build();
+
+        ResourcePolicy resourcePolicyOfEPerson1 = ResourcePolicyBuilder.createResourcePolicy(context)
+                .withDspaceObject(community).withAction(Constants.ADD).withPolicyType(ResourcePolicy.TYPE_CUSTOM)
+                .withUser(eperson1).build();
+
+        ResourcePolicy resourcePolicyOfEPerson2 = ResourcePolicyBuilder.createResourcePolicy(context)
+                .withDspaceObject(community2).withAction(Constants.REMOVE)
+                .withPolicyType(ResourcePolicy.TYPE_CUSTOM).withUser(eperson2).build();
+
+        context.restoreAuthSystemState();
+
+        String authToken = getAuthToken(eperson1.getEmail(),"qwerty01");
+
+        getClient(authToken).perform(get("/api/authz/resourcepolicies/search/eperson")
+                .param("uuid",eperson1.getID().toString()))
+        .andExpect(status().isOk())
+        .andExpect(content().contentType(contentType))
+        .andExpect(jsonPath("$._embedded.resourcepolicies", Matchers.contains(
+                 ResoucePolicyMatcher.matchResourcePolicy(resourcePolicyOfEPerson1))))
+        .andExpect(jsonPath("$._embedded.resourcepolicies",
+                Matchers.not(is(ResoucePolicyMatcher.matchResourcePolicy(resourcePolicyOfEPerson2)))))
+        .andExpect(jsonPath("$._links.self.href", Matchers.containsString("api/authz/resourcepolicies/search/eperson")))
+        .andExpect(jsonPath("$.page.totalElements", is(1)));
+    }
+
+
+    @Test
+    public void searchResoucesPolicyByEpersonUuidAndResourceUuidTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+
+        EPerson eperson1 = EPersonBuilder.createEPerson(context).withEmail("myemail@mail.com").withPassword("qwerty01")
+                .build();
+
+        Community community = CommunityBuilder.createCommunity(context).withName("My community").build();
+        Collection collection = CollectionBuilder.createCollection(context, community).withName("My collection")
+                .build();
+
+        ResourcePolicy resourcePolicyOfCommunity = ResourcePolicyBuilder.createResourcePolicy(context)
+                .withDspaceObject(community).withAction(Constants.READ).withPolicyType(ResourcePolicy.TYPE_CUSTOM)
+                .withUser(eperson1).build();
+
+        ResourcePolicy secondResourcePolicyOfCommunity = ResourcePolicyBuilder.createResourcePolicy(context)
+                .withDspaceObject(community).withAction(Constants.REMOVE).withPolicyType(ResourcePolicy.TYPE_CUSTOM)
+                .withUser(eperson1).build();
+
+        ResourcePolicy resourcePolicyOfCollection = ResourcePolicyBuilder.createResourcePolicy(context)
+                .withDspaceObject(collection).withAction(Constants.REMOVE)
+                .withPolicyType(ResourcePolicy.TYPE_SUBMISSION).withUser(eperson1).build();
```

Comment:
usually resourcepolicy that apply to a collection haven't a type so it would be better use a more "real-world" example here removing the policyType